Ginger/Ginger/Actions/ActionConversion/SelectBusinessFlowWzardPage.xaml.cs

```diff
@@ -74,6 +74,7 @@ private void SetActivitiesSelected()
         {
             foreach (var businessFlow in ListOfBusinessFlow)
             {
+                businessFlow.BusinessFlow.CreateBackup();
```

Comment:
1- i don't think that under method called SetActivitiesSelected() we should do backup, it is not related, better call the method "PrepareBFsForConversion()"
2- why you iterate over all Activities and setting the SelectedForConversion = true, can't you as default convert all Activities if running in multi BF's conversion mode?

Reply:
1 - if we want to create new method then will again have to iterate it again and will have to do backup, so just not to iterate it again added that line in that method.
2 - this is done since there is existing code in utils which is dependent on the "SelectedForConversion" flag, so will have to modify there as will this will require some tricky change so to avoid that added this method to set the flag for activities